frontend/src/main-components/MyFoldersContent.js

```diff
@@ -9,11 +10,54 @@ const MyFoldersContent = (props) => {
     border-radius: 1rem;
     background-color: white;
     margin: 1%;
+    flex-direction: column;
   `;
+  const folderStyle = {
+    display: "flex",
+    flexFlow: "row wrap",
+    justifyContent: "space-around",
+    alignItems: "start",
+  };
+
+  let folders;
+
+  //Fetch data from the UserFolders servlets to get the user folders
+  //Currently there is no way to get the userid of the user
+  try {
+    fetch(`/userfolder?userKey=${userKey}`, { method: "GET" })
+      .then((result) => result.json())
+      .then((data) => {
+        console.log(data);
+        folderData = data;
+        const folders = folderData.map((folder) => (
+          <Folder
+            folderKey={folder.folderKey}
+            name={folder.folderName}
+            language={folder.folderDefaultLanguage}
+            key={folder.folderKey}
+          />
```

Comment:
Shouldn't there only be one "folder.folderKey"?

Reply:
I ran into some compile errors while passing in key as a prop, so I added the folderKey prop as way to circumvent that.